TrkDiag/inc/TimeClusterInfo.hh

```diff
@@ -64,6 +67,9 @@ namespace mu2e {
     Int_t	_nceclust; // # of conversion electron hits found in clusters
     Float_t	_time; // average time of CE hits (doesn't include drift!)
     XYZVec	_pos; // average position of cluster
+    Float_t _mcposx; // average position, S Middleton
```

Comment:
Please use XYZVec instead of separate floats

Reply:
will either do or remove if not needed any more once I rejig above